tests/.coveragerc

```diff
@@ -1,8 +1,16 @@
 [run]
-omit = */django/contrib/*/tests*,*/django/utils/autoreload.py
+branch = True
+omit =
+    */django/contrib/*/tests*
```

Comment:
I don't think this line is relevant anymore as tests have been moved out of contrib.